tests/vm_test.go

```diff
@@ -977,6 +977,10 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				})
 
 				It("[test_id:4119]should migrate a running VM", func() {
+					nodes := tests.GetAllSchedulableNodes(virtClient)
+					if len(nodes.Items) < 2 {
+						Skip("Migration tests require at least 2 nodes")
```

Comment:
What cause this one to surface only now?
How will we know if this is not skipped unintentionally?

Reply:
Just run the tests locally on your machine with only one node and you will see it fail. In CI we always run with two nodes. It has the same risk like all the other tests which are skipped like this for the same reason.

Reply:
Here a WIP PR, based on this one which can eventually address these things to a degree: https://github.com/kubevirt/kubevirt/pull/4201.

Reply:
I always run locally with two nodes, so I guess that is why I never saw this.
The pre-flight checks should do the job I guess, especially in a conformance test.